schema/me/update_collector_profile.js

```diff
@@ -1,7 +1,31 @@
 import { CollectorProfileFields } from "./collector_profile"
-import { GraphQLBoolean, GraphQLString, GraphQLList } from "graphql"
+import { GraphQLBoolean, GraphQLString, GraphQLList, GraphQLEnumType } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 
+export const IntentsType = new GraphQLEnumType({
+  name: "Intents",
+  values: {
+    BUY_ART_AND_DESIGN: {
+      value: "buy art & design",
+    },
+    SELL_ART_AND_DESIGN: {
+      value: "sell art & design",
+    },
+    RESEARCH_ART_PIECES: {
```

Comment:
This says `PIECES`, but the `value` has `prices`. Intentional?

Reply:
lol not at all!